IRuettae/DebugApps/IRuettae.Evaluator/Program.cs

```diff
@@ -146,7 +146,7 @@ private static void EvaluateAlgorithm(string[] args)
                                 timelimit /= fastFactor;
                                 goto case Algorithms.LocalSolver;
                             case Algorithms.LocalSolver:
-                                solver = new IRuettae.Core.LocalSolver.Solver(input, false, false);
+                                solver = new IRuettae.Core.LocalSolver.Solver(input, 0.1,0.8,false);
```

Comment:
Könnte man auch mit Objekt lösen, da man in der WebApi eh eine Klasse/Struktur machen müsste. Aber passt schon.